calico_cni_k8s_test.go

```diff
@@ -309,6 +309,93 @@ var _ = Describe("CalicoCni", func() {
 					}))
 				})
 			})
+
+			Context("using ipAddrs annotation to assign IP address to a pod, through Calico IPAM", func() {
```

Comment:
Discussed this offline. Seems like an issue with our testing framework where it calls netlink only with IPv4 address. Not worth spending time on it for now, but would be good to fix that for future tests.